apps/hash-api/src/graph/ensure-system-graph-is-initialized/migrate-ontology-types/migrations/015-add-flow-types.dev.migration.ts

```diff
@@ -239,6 +249,72 @@ const migrate: MigrationFunction = async ({
     },
   );
 
+  /**
+   * Step 3: create a `Incurred In` link entity type, and update
+   * the `Usage Record` entity type to have it as an outgoing link to a `Flow`.
+   */
+
+  const incurredInLinkEntityType = await createSystemEntityTypeIfNotExists(
+    context,
+    authentication,
+    {
+      entityTypeDefinition: {
+        allOf: [linkEntityTypeUrl],
+        title: "Incurred In",
+        description: "Something that was incurred by something else.",
```

Comment:
Consistency (don't really mind `Incurred By` or `Incurred During` either)
```suggestion
        description: "Something that was incurred in something else.",
```